server/pages/index.jade

```diff
@@ -110,25 +110,45 @@ html(lang=lang, dir=isRTL ? 'rtl' : 'ltr', class=isFluidWidth ? 'is-fluid-width'
 				}
 			})();
 
-		if user
+		if supportUser
+			script.
+				window.supportUser = {
+					readyScripts: []
+				};
+		if user && !supportUser
 			script(type='text/javascript')!='var currentUser = ' + sanitize.jsonStringifyForHtml( user )
 		if app
 			script(type='text/javascript')!='var app = ' + sanitize.jsonStringifyForHtml( app )
-		if initialReduxState
+		if initialReduxState && !supportUser
```

Comment:
For reference [`server-side-rendering`](https://github.com/Automattic/wp-calypso/blob/f9f0d81e41d9d3f8d33e72772b52ff5239a112af/config/wpcalypso.json#L85) is the relevant feature flag, if we ever want to make sure server rendering does not occur when support user is active.